packages/conf-flint/conf-flint.3.0/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+maintainer: "bobot"
+homepage: "http://flint.org"
+bug-reports: "https://github.com/flintlib/flint2.git"
+license: "LGPL-2.1-only"
+build: [
+  ["sh" "-exc" "cc -c $CFLAGS -I/usr/local/include test.c"] {os != "macos" & os != "win32"}
+  [
+		"sh"
+		"-exc"
+		"$(ocamlc -config-var c_compiler) -c $CFLAGS -I/usr/local/include test.c"
+	] {os = "win32" & os-distribution = "cygwinports"}
+  [
+    "sh"
+    "-exc"
+    "cc -c $CFLAGS -I/opt/homebrew/include -I/opt/local/include -I/usr/local/include test.c"
+  ] {os = "macos"}
+]
+depexts: [
+  ["libflint-dev"] {os-family = "debian"}
+  ["libflint-dev"] {os-family = "ubuntu"}
+  ["flint"] {os = "macos" & os-distribution = "homebrew"}
+  ["flint"] {os-distribution = "macports" & os = "macos"}
+  ["flint" "flint-devel"] {os-distribution = "centos"}
+  ["flint" "flint-devel"] {os-distribution = "fedora"}
+  ["flint" "flint-devel"] {os-distribution = "ol"}
+  ["flint-dev"] {os-distribution = "alpine"}
+  ["flint-devel"] {os-family = "suse"}
+  ["libflint-devel"] {os = "win32" & os-distribution = "cygwinports"}
+  ["flint"] {os-distribution = "nixos"}
```

Comment:
```suggestion
  ["flint"] {os-distribution = "nixos"}
  ["flint"] {os-distribution = "arch"}
```
(note: I'm unsure whether it's best to use `os-distribution` or `os-family` as both examples seem to exist)

Reply:
I don't know if that's relevant or not, but at least on `arch`, the include files are in `/usr/include` and not `/usr/local/include`